pkg/handler/eval.go

```diff
@@ -39,7 +39,8 @@ func (e *eval) PostEvaluation(params evaluation.PostEvaluationParams) middleware
 			ErrorMessage("empty body"))
 	}
 
-	evalResult := EvalFlag(*evalContext)
+	flag := LookupFlag(*evalContext)
+	evalResult := EvalFlagWithContext(flag, *evalContext)
```

Comment:
since the function EvalFlag is still there and `flag` variable is not used below the line, I think you can keep `evalResult := EvalFlag(*evalContext)`